docs/tools/firmware.sh

```diff
@@ -43,54 +44,34 @@ case "$os" in
 				cp /usr/share/firmware/wifi/C-4364__s-B2/${txcapblob} "/Volumes/${EFILABEL}/brcmfmac4364b2-pcie.txcap_blob"
 			fi
 		fi
-		echo "Copying this script to EFI"
-		cd - >/dev/null
-		cp "$0" "/Volumes/${EFILABEL}/firmware.sh"|| (echo -e "\nFailed to copy script.\nPlease copy the script manually to the EFI partition using Finder\nMake sure the name of the script is firmware.sh in the EFI partition\n" && echo && read -p "Press enter after you have copied" && echo)
+		echo "Copying this script to the ESP"
+		cp "$0" "/Volumes/${EFILABEL}/firmware.sh"|| (echo -e "\nFailed to copy script.\nPlease copy the script manually to the EFI partition using Finder\n" && echo && read -p "Press enter after you have copied" && echo)
```

Comment:
Why did you remove the "make sure the name of the script is firmware.sh" when you have asked the user to run /tmp/apple-wifi-efi/firmware.sh in the following commands.